examples/image-classification-web/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+authors = ["Dilshod Tadjibaev (@antimora)"]
+edition = "2021"
+license = "MIT OR Apache-2.0"
+name = "image-classification-web"
+publish = false
+version = "0.10.0"
+
+[lib]
+crate-type = ["cdylib"]
+
+[features]
+default = []
+
+[dependencies]
+burn = { path = "../../burn", default-features = false, features = [
+    "ndarray-no-std",
+    "wgpu",
+] }
+
+burn-candle = { path = "../../burn-candle", version = "0.10.0", default-features = false }
+
+getrandom = { workspace = true, features = ["js"] }
```

Comment:
Is it necessary here? With the burn-common update, I think it should not be required in application code.